### PR TITLE
Update Compose samples

### PR DESCRIPTION
**Pull request by joyl1216**
## 📃 Description
Update Compose samples with the updated libraries

## 🤔 Motivation and Context
Keep the Compose libraries up to date to include the bug fix in the samples

## 🧪 How Has This Been Tested?
Manually test it on both the emulator and device.

## ⚒️ Does the PR contain new tests or refactored old ones?
No

## 📦 Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement to a current functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Bug fix (non-breaking change which fixes an issue. Please add issue link here too).

## ✅ Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] This PR contains new tests that cover the new code.
- [ ] This PR refactor previous tests to cover the new code.
- [ ] This PR is part of a set of PRs that build a bigger feature. If so, please, add here links to previously merged or open PRs.

### CODE REVIEW

File: ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/ListPane.kt [hunk 1]
```diff
@@ -79,27 +75,23 @@ private fun ListActions() {
...
-    isDualMode: Boolean,
     selectedImageIndex: Int,
     updateImageIndex: (Int) -> Unit,
-    lazyListState: LazyListState,
 ) {
     LazyColumn(
         modifier = Modifier
             .fillMaxSize()
             .testTag(stringResource(R.string.gallery_list)),
-        state = lazyListState,
```
Comment (khalp) [inline ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/ListPane.kt:91]: I'm just curious - why did you remove the lazy list state?

File: DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/view/RestaurantView.kt [hunk 2]
```diff
@@ -104,7 +104,7 @@ private fun RestaurantActionButton() {
 }
 
 @Composable
-fun RestaurantView(viewWidth: Int, selectedIndex: Int, updateSelectedIndex: (Int) -> Unit, isDualScreen: Boolean) {
+fun RestaurantView(viewWidth: Int, selectedIndex: Int, updateSelectedIndex: (Int) -> Unit) {
```
Comment (khalp) [inline DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/view/RestaurantView.kt:107]: I think one of the DualView UI tests needs to be updated after this change

image

File: ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/ListPane.kt (hunk 1 above)
Comment (joyl1216) [reply ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/ListPane.kt:91]: I didn't see any customized need for creating our own `lazyListState` here. By default, `LazyColumn` provides a `LazyListState` via `state: LazyListState = rememberLazyListState()`.

File: DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/view/RestaurantView.kt (hunk 2 above)
Comment (joyl1216) [reply DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/view/RestaurantView.kt:107]: Good catch

File: ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/ListPane.kt (hunk 1 above)
Comment (khalp) [reply ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/ListPane.kt:91]: Ahh ok makes sense, thank you for the explanation!

**Review by khalp — APPROVED** [review]
